frontend/app/src/store/utils.ts

```diff
@@ -62,68 +58,66 @@ export async function fetchAsync<S, T extends TaskMeta, R>(
       display: true
     });
   }
-  setStatus(Status.LOADED, section, status, commit);
+  setStatus(Status.LOADED, section);
 }
 
 export function showError(description: string, title?: string) {
+  const { setMessage } = useMainStore();
   const message = {
     title: title ?? i18n.t('message.error.title').toString(),
     description: description || '',
     success: false
   };
-  store.commit('setMessage', message);
+  setMessage(message);
 }
 
 export function showMessage(description: string, title?: string): void {
+  const { setMessage } = useMainStore();
   const message: Message = {
     title: title ?? i18n.t('message.success.title').toString(),
     description,
     success: true
   };
-  store.commit('setMessage', message);
+  setMessage(message);
 }
 
-export const setStatus: (
-  newStatus: Status,
-  section: Section,
-  status: (section: Section) => Status,
-  commit: Commit
-) => void = (newStatus, section, status, commit) => {
-  if (status(section) === newStatus) {
+export const getStatus = (section: Section) => {
+  const { getStatus } = useMainStore();
+  return unref(getStatus(section));
+};
+
+export const setStatus: (newStatus: Status, section: Section) => void = (
```

Comment:
would it be better to also create one for `resetStatus`, since there are pretty much usage for it

Reply:
sure it should be easy to add